### ISSUE TITLE
kubevirt field failed to be recognized when creating a vcluster

### ISSUE THREAD

**Issue by jingyan112** [loft-sh/vcluster#2124]
### What happened?

# vcluster --version
vcluster version 0.20.0

root@vcluster-ThinkPad-T14p-Gen-1:/home/vcluster# cat values.yaml
controlPlane:
  distro:
    k8s:
      enabled: true
      version: "v1.30.4"
      apiServer:
        enabled: true
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.aliyuncs.com/google_containers
          repository: kube-apiserver
          tag: "v1.30.4"
      controllerManager:
        enabled: true
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.aliyuncs.com/google_containers
          repository: "kube-controller-manager"
          tag: "v1.30.4"
      scheduler:
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.k8s.io
          repository: "kube-scheduler"
          tag: "v1.30.2"
      env: []
      securityContext: {}
sync:
  toHost:
    storageClasses:
      enabled: true
    persistentVolumes:
      enabled: true
    persistentVolumeClaims:
      enabled: true
integrations:
  kubeVirt:
    enabled: true
    webhook:
      enabled: true
    sync:
      dataVolumes:
        enabled: true
      virtualMachines:
        enabled: true
      virtualMachineInstances:
        enabled: true
      virtualMachinePools:
        enabled: true
      virtualMachineClones:
        enabled: true
      virtualMachineInstanceMigrations:
        enabled: true

# vcluster create vcluster2 --namespace vcluster2 -f values.yaml
17:04:24 info Creating namespace vcluster2
17:04:25 fatal invalid config: error unmarshaling JSON: while decoding JSON: json: unknown field "kubeVirt"

### What did you expect to happen?

According to the template yaml: https://github.com/loft-sh/vcluster/blob/main/chart/values.yaml
And the doc in official website: https://www.vcluster.com/docs/vcluster/next/configure/vcluster-yaml/integrations/#integrations-kubeVirt, kubevirt field should be supported.

### How can we reproduce it (as minimally and precisely as possible)?

See the commands above

### Host cluster Kubernetes version

<details>

```console
$ kubectl version
Client Version: v1.30.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.30.4
```

</details>

### vcluster version

<details>

```console
$ vcluster --version
vcluster version 0.20.0
```

</details>

### VCluster Config

<details>

```
# My vcluster.yaml / values.yaml here
controlPlane:
  distro:
    k8s:
      enabled: true
      version: "v1.30.4"
      apiServer:
        enabled: true
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.aliyuncs.com/google_containers
          repository: kube-apiserver
          tag: "v1.30.4"
      controllerManager:
        enabled: true
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.aliyuncs.com/google_containers
          repository: "kube-controller-manager"
          tag: "v1.30.4"
      scheduler:
        command: []
        extraArgs: []
        imagePullPolicy: ""
        image:
          registry: registry.k8s.io
          repository: "kube-scheduler"
          tag: "v1.30.2"
      env: []
      securityContext: {}
sync:
  toHost:
    storageClasses:
      enabled: true
    persistentVolumes:
      enabled: true
    persistentVolumeClaims:
      enabled: true
integrations:
  kubeVirt:
    enabled: true
    webhook:
      enabled: true
    sync:
      dataVolumes:
        enabled: true
      virtualMachines:
        enabled: true
      virtualMachineInstances:
        enabled: true
      virtualMachinePools:
        enabled: true
      virtualMachineClones:
        enabled: true
      virtualMachineInstanceMigrations:
        enabled: true
```

</details>

**Comment by deniseschannon** [comment]
You need to try the alpha versions for 0.21.0 as this functionality is not available in 0.20.0.

**Comment by jingyan112** [comment]
yeah, it works on 0.21.0.
